src/components/RoutePage/Amenity.js

```diff
@@ -5,7 +5,7 @@ import Icon from "../Common/Icon";
 const Amenity = ({ routeName, iconName, name }) => {
     return (
         <A
-            href={`/route/${routeName}`}
```

Comment:
Why this route is removed

Reply:
This is a link to the same page and that page has not been designed yet and the `routeName`'s casing is not consistent throughout the JSON. So, since its a link to the same page I replaced it with `#` instead of doing `toLowerCase()`